### PR TITLE
Added support for KeePassXC

### PR DESCRIPTION
**Pull request by harens**
Added support for the password manager [KeePassXC](https://keepassxc.org/). This is based on the [KeePassX config file](https://github.com/lra/mackup/blob/master/mackup/applications/keepassx.cfg).

### CODE REVIEW

**Review by treee111 — APPROVED** [review]
I tested the changes by applying commit c58bb1 to lra/master and make locally on my Mac.
Works and the relevant files are updated  👍

File: mackup/applications/keepassxc.cfg [hunk 1]
```diff
@@ -0,0 +1,6 @@
+[application]
+name = KeePassXC
+
+[configuration_files]
+Library/Preferences/org.keepassxc.keepassxc.plist
+Library/Application Support/keepassxc/keepassxc.ini
```
Comment (lra) [inline mackup/applications/keepassxc.cfg:6]: missing newline

**Review by lra — APPROVED** [review]
Thx @treee111 helps a lot to know someone else actually tested it and approved.

**Review by lra — APPROVED** [review]

File: mackup/applications/keepassxc.cfg (hunk 1 above)
Comment (lra) [reply mackup/applications/keepassxc.cfg]: https://github.com/lra/mackup/pull/1611/commits/b020ea3bf6b31079984d4a5cbd5447edd71b559d

**Comment by harens** [comment]
Thank you @lra for merging this, and thank you @treee111 for testing the changes!

**Comment by paxperscientiam** [comment]
I'm actually running into an issue with this. Any thoughts? Thanks

Here's the output from python:
```shell
Traceback (most recent call last):
  File "/Users/ramos/Library/Python/3.7/bin/mackup", line 33, in <module>
    sys.exit(load_entry_point('mackup==0.8.32', 'console_scripts', 'mackup')())
  File "/Users/ramos/Library/Python/3.7/lib/python/site-packages/mackup/main.py", line 91, in main
    app = ApplicationProfile(mckp, app_db.get_files(app_name), dry_run, verbose)
  File "/Users/ramos/Library/Python/3.7/lib/python/site-packages/mackup/appsdb.py", line 141, in get_files
    return self.apps[name]["configuration_files"]
KeyError: 'keepassxc'
```

**Comment by lra** [comment]
It's not released yet, but you can add this file (https://github.com/lra/mackup/blob/master/mackup/applications/keepassxc.cfg) to your current install: https://github.com/lra/mackup/tree/master/doc#add-support-for-an-application-or-any-file-or-directory

**Comment by paxperscientiam** [comment]
Worked beautifully. Thanks!

**Pull request merged**